sample-error-response-v2.yaml

```diff
@@ -0,0 +1,72 @@
+openapi: '3.0.0'
+info:
+  description: This specification is only for sample error response for API v2.
+  version: '1.0'
+  title: Sample Error Response
+servers:
+  - url: 'http://sample-service'
+paths:
+  /v2/sample/error:
+    get:
+      summary: Sample Request for Error Response
+      responses:
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+components:
+  responses:
+    InternalServerError:
+      description: Server error. This should happen rarerly.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+          example:
+            timestamp: 2022-06-17T14:07:17Z
+            trace_id: aswo3f-a202lfso-312123sld-1230dd1
+            errors:
+              - title: Internal error.
+                details: Your request cannot be completed. Please try again later.
+                type: https://developer.personio.de/reference/errors#sample_service.internal_error
+  schemas:
+    ErrorResponse:
+      description: Invalid request response.
+      type: object
+      properties:
+        timestamp:
+          description: The timestamp.
```

Comment:
Question: Should we be more precise with this description? Maybe we can add that this timestamp is for the request that was executed?

For example: The timestamp of the executed request